.ci/install_qemu.sh

```diff
@@ -201,29 +60,27 @@ main() {
 		esac
 	done
 
+	export qemu_type
 	case "${qemu_type}" in
-		tdx)
-			CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.tdx.tag")
-			QEMU_REPO_URL=$(get_version "assets.hypervisor.qemu.tdx.url")
-			qemu_latest_build_url="${jenkins_url}/job/kata-containers-2.0-qemu-tdx-$(uname -m)/${cached_artifacts_path}"
-			;;
 		vanilla)
-			CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.version")
-			QEMU_REPO_URL=$(get_version "assets.hypervisor.qemu.url")
-			qemu_latest_build_url="${jenkins_url}/job/kata-containers-2.0-qemu-$(uname -m)/${cached_artifacts_path}"
+			qemu_type="qemu"
 			;;
 		*)
-			info "qemu type '${qemu_type}' not supported"
-			usage 1
+			die_unsupported_qemu_type "$qemu_type"
 			;;
 	esac
 
-	# Strip "https://" to clone into $GOPATH
-	gopath_qemu_repo=${QEMU_REPO_URL/https:\/\//}
-	# These variables are used by static-build scripts
-	export qemu_version="${CURRENT_QEMU_VERSION}"
-	export qemu_repo="${QEMU_REPO_URL}"
-	run
+	case ${QEMU_ARCH} in
+		"ppc64le")
+			build_and_install_qemu
```

Comment:
@fidencio, one question here. With this change, the `qemu-ppc64` binary will be installed in `/usr/bin`. However the default path that is being used in the config `/opt/kata/share/defaults/kata-containers/configuration.toml` is
```
[hypervisor.qemu]
path = “/opt/kata/bin/qemu-system-ppc64”
```
If my understanding is correct, this should also be changed right?

Reply:
You're correct.  Ideally arch specific script should rexpect the ${PREFIX} being used.

I will adjust that, thanks for pointing this out!